src/codeu/chat/client/Controller.java

```diff
@@ -65,14 +65,15 @@ public Message newMessage(Uuid author, Uuid conversation, String body) {
   }
 
   @Override
-  public User newUser(String name) {
+  public User newUser(String name, String password) {
 
```

Comment:
I know this isn't yours but these new lines after the method signature are triggering.